packages/api/src/command/medical/document/document-query.ts

```diff
@@ -88,6 +90,7 @@ export async function queryDocumentsAcrossHIEs({
     downloadProgress: { status: "processing" },
     requestId,
     reset: true,
+    cxDocumentRequestMetadata,
```

Comment:
stuck with updating db here. optional flag makes it compatible. I think this is the simplest approach as opposed to creating a new dedicated fxn to update the cxDocumentRequestMetadata